bin/sendAlertStream.py

```diff
@@ -21,6 +22,29 @@ def load_stamp(file_path):
     return cutout_dict
 
 
+@asyncio.coroutine
+def delay(wait_sec, function):
+    """Sleep for a given time before calling a function.
+    """
+    print('delay starting: {}'.format(time.time()))
+    yield from asyncio.sleep(wait_sec)
+    print('delay done sleeping: {}'.format(time.time()))
+    return function()
+    print('delay finished: {}'.format(time.time()))
+
+
+@asyncio.coroutine
+def schedule_delays(eventloop, function, maxcounts):
+    counter = 0
+    while counter < maxcounts:
+        seconds_to_minute = 39 - (time.time() % 39)  # on the 39th second
```

Comment:
It's probably worth making 39 a variable somewhere (maybe a kwarg to the function?), instead of a magic parameter. Coudl then use it to test scaling and such.